### PR TITLE
add toMillis method

### PR DESCRIPTION
**Pull request by maxymshg**
Patch for #246 .

I'll saw you comment, and plan to add test(s) later (probably tomorrow). Please note that by some reason a lot of existing tests are failed on my machine.

Could you please meanwhile briefly look on patch and provide a feedback what else should be done?

### CODE REVIEW

File: docs/moment.md [hunk 1]
```diff
@@ -137,11 +137,12 @@ See the [formatting guide](formatting.html) for more about the string-outputting
...
 | RFC 2822             |                | `toRFC2822()`    |                                                                                                                                                   |
 | HTTP date string     |                | `toHTTP()`       |                                                                                                                                                   |
 | JS Date              | `toDate()`     | `toJSDate()`     |                                                                                                                                                   |
 | Epoch time           | `valueOf()`    | `valueOf()`      |                                                                                                                                                   |
+| Time in milliseconds |                | `toMillis()`     |                                                                                                                                                   |
```
[the remark this answers is not in the document]
Comment (maxymshg) [reply docs/moment.md]: A bit unclear, should I update second column with reference to Epoch time or just left comment in Notes column?

**Comment by maxymshg** [comment]
`scripts/test` failed for me, maybe it's because I'm doing this on WSL

File: docs/moment.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (icambron) [reply docs/moment.md]: I'm saying:

```
| Epoch time | `valueOf` | `valueOf` or `toMillis` | 
```

**Comment by maxymshg** [comment]
@icambron, I complete all changes to pass checks. Could you suggest what else should be added/changed?

**Comment by icambron** [comment]
Looks good, thanks!

**Comment by icambron** [comment]
I'll release this soon

**Pull request merged**